resources/templates/md/posts/2019-05-18-karma-feature-announcement.md

```diff
@@ -0,0 +1,37 @@
+{:title "Karma Feature Announcement"
+ :layout :post
```

Comment:
I just added support for additional meta for better link unfurling. Can you add `:summary` and `:image` keys? Example [here](https://github.com/yetibot/yetibot.github.io/blob/source/resources/templates/md/posts/2019-05-18-yetis-at-clojure-north.md#L4-L5).